setup/docker/README.txt

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Yes, sure. Create a link on website with content of this docker/README.txt and we and update ./setup/README.txt